### ISSUE TITLE
PHP (Tutorial) not working

### ISSUE THREAD

**Issue by Hanfrey** [fnproject/fn#495]
Hi,

i tried to execute your PHP code, but it does not work for me, the GO and node example just works fine. 
```

# Initialize your function creating a func.yaml file
fn init --name hello-php

# Test your function. 
# This will run inside a container exactly how it will on the server. It will also install and vendor dependencies from Gemfile
fn run

Building image hanfrey/hello-php:0.0.1
Sending build context to Docker daemon   5.12kB
Step 1/5 : FROM funcy/php:dev as build-stage
dev: Pulling from funcy/php
...
Digest: sha256:cac7bfa17991a48267215e33344acb53fcdccd9c6a78172adcafb9df205c97b8
Status: Downloaded newer image for funcy/php:dev
 ---> 7a88b478f08d
Step 2/5 : WORKDIR /function
 ---> 41cd7383b1be
Removing intermediate container 4b7b7f0a4f67

****Step 3/5 : FROM
FROM requires either one or three arguments**
ERROR: error running docker build: exit status 1**

```

Normally in the step should be the funcy/php image, no?

My func.yaml:

```
name: hello-php
version: 0.0.1
runtime: php
entrypoint: php func.php
```

fn version 0.4.16 .  also tried a 3.8 version

**Comment by carimura** [comment]
thanks @Hanfrey .. CC @michael-w-williams

**Comment by michael-w-williams** [comment]
I will take a look today.

**Comment by michael-w-williams** [comment]
Reproduced the error on my system. Happening during the build phase. I getting the same message:

```
$ fn build
Building image hello-php:0.0.1
Sending build context to Docker daemon  4.096kB
Step 1/5 : FROM funcy/php:dev as build-stage
 ---> 7a88b478f08d
Step 2/5 : WORKDIR /function
 ---> Using cache
 ---> 081ad227977e
Step 3/5 : FROM
FROM requires either one or three arguments
ERROR: error running docker build: exit status 1
```
Checked DockerHub and the entry for the image is there along with the dev tag. Dockerfile images exist in github too. No help from Ms Google, so will check with the team on this one.

**Comment by zootalures** [comment]
This was a bug in the php runtime - I've fixed this  as part of fnproject/cli#90
